_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -763,372 +763,2410 @@ Terraform::
 
-[[cleanup_expired_certs_deployment]]
-=== Automatically remove expired SSL/TLS certificates from IAM
-Next, use the link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/cleanup-expired-certs/README.adoc[`cleanup-expired-certs`]
-module to deploy a Lambda function to remove all expired SSL/TLS certificates from AWS IAM. This module will add all of the relevant IAM resources needed to
-periodically execute the Lambda function and optionally, report metrics to CloudWatch. Be sure to deploy this module into each of your AWS accounts.
+. **Configure the Auto Deploy IAM role**. We also create an `auto-deploy` IAM role that can be assumed by a CI server
+in the shared-services account to do deployments. This role will have the permissions specified in
+`auto_deploy_permissions`.
 
-If you're using Terragrunt, then you can invoke the standalone module directly by following the pattern outlined in
-<<wrapper_code_terragrunt>> and creating a `terragrunt.hcl` file with the following contents:
+. **Configure cross-account IAM roles**. We then specify which other accounts are allowed to assume the IAM roles in
+this account. For the most part, we grant all permissions to the security account, so that by assigning users to IAM
+groups in that account, you'll be able to access IAM roles in all the other child accounts.
 
-.infrastructure-live/<your account>/_global/cleanup-expired-certs/terragrunt.hcl
+Configure your Terraform backend:
+
+.infrastructure-live/stage/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
-terraform {
-  # Make sure to replace <VERSION> in this URL with the latest terraform-aws-cis-service-catalog release
-  source = "git::ssh://git@github.com/gruntwork-io/terraform-aws-cis-service-catalog.git//modules/cleanup-expired-certs?ref=<VERSION>"
-}
-
-# Include all settings from the root terragrunt.hcl file
 include {
   path = find_in_parent_folders()
 }
+----
 
-inputs = {
-  schedule_expression                = "rate(1 hour)"
-  report_cloudwatch_metric           = true
-  report_cloudwatch_metric_namespace = "custom/cis"
-  report_cloudwatch_metric_name      = "cleanup-expired-iam-certs-count"
-}
+Just as with the logs and security accounts, you're going to use the `OrganizationAccountAccessRole` IAM role created by
+`account-baseline-root` to authenticate to the stage account and all other child accounts. There are many ways to
+https://blog.gruntwork.io/a-comprehensive-guide-to-authenticating-to-aws-on-the-command-line-63656a686799[assume an IAM role on the CLI];
+for this guide, we're going to keep using `aws-vault`.
+
+Add a new `profile` entry in `~/.aws/config` for your stage account that uses the `root-iam-user` as the
+`source_profile`:
+
+[source,text]
+----
+[profile stage-from-root]
+role_arn=arn:aws:iam::<STAGE_ACCOUNT_ID>:role/OrganizationAccountAccessRole
+source_profile=root-iam-user
 ----
 
-[[iam_access_analyzer_deployment]]
```

Comment:
This section has been removed but is still being linked to from other parts of the document. We should reconcile that.

Reply:
Hm, looking at that - it's just that we're no longer deploying that on its own. We're now using the baselines to deploy that. So maybe just remove the refs to this link & make sure the sentences make sense.

Reply:
@ina-stoyanova @infraredgirl I spoke to Ana and we agreed to resolve this.